lib/ansible/module_utils/scaleway.py

```diff
@@ -32,6 +49,69 @@ def ok(self):
         return self.status_code in (200, 201, 202, 204)
 
 
+class Scaleway(object):
```

Comment:
Should not `Scaleway`, `_ScalewayApi` and `ScalewayAPI` be only one class ?

Reply:
`_ScalewayAPI` has been removed. The other two exists during the transition phase while we rewrite `scaleway_volume.py`, `scaleway_compute.py` and `scaleway_ssh_key.py` to match the latest version of the class.